tools/pip_package/setup.py

```diff
@@ -40,7 +40,6 @@
     long_description=long_description,
     py_modules=py_modules,
     packages=find_packages(),
-    install_requires=["typing>=3.6.2"],
```

Comment:
Same as previous comment. `typing` module is need to complete codes at some cases.

Reply:
If my understanding written in `requirements.txt` comment is correct, we can delete this line.